learn/wavemaker-release-notes/v10-9-2.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "WaveMaker 10.9.2 - Release date: 01 November 2021"
+id: ""
+sidebar_label: "Release v10.9.2"
+---
+*WaveMaker released a new update version with some bug fixes.*
+
+## Bug Fixes
+
+- The improved search capability of the wizard widget in the widget panel; added new keywords, including step, stepper, and wizard. 
+- Fixed an issue related to the setFilter method of the CRUD variable showing console errors.
+- Fixed an issue related to the date picker inside a dialog that did not show correctly.
+- Fixed an issue related to the generated-angular-app folder in WaveMaker Studio. 
```

Comment:
Please change this to:
"The generated Angular app can now be viewed in studio while working on the project."

Reply:
Done.